### DISCUSSION TITLE
[Feature]: Turn Xpetrified orbs into XP fluid

### DISCUSSION THREAD

**Discussion by Gogo64pro** [AllTheMods/ATM-10#2763] [Modpack Content]
### Feature Details

With the forges you need XP fluid and you can get a lot of Xpetrified orbs from bees, but there's no way to convert them so you're stuck with a lot of the Xpetrified orbs

**Comment by Aidenboss434** [comment]
Maybe it could be a Create: Enchantment Industry Grinding Recipe
![image](https://github.com/user-attachments/assets/c8195f9c-eed2-4508-b6b6-80ed83b28f5d)

Example Kubejs Script:
```
ServerEvents.recipes(event => {
  event.custom({
    type: "create_enchantment_industry:grinding",
    ingredients: [
      {
        item: "forbidden_arcanus:xpetrified_orb"
      }
    ],
    results: [
      {
        amount: 9,
        id: "create_enchantment_industry:experience"
      }
    ]
  }).id("allthemods:grinding/xpetrified_orb");
});
```
